### PR TITLE
add settings link for media site module to site attribution

### PR DESCRIPTION
**Pull request by andytuba**
screen shot 2016-05-18 at 2 37 59 pm
site settings link

### CODE REVIEW

**Comment by erikdesjardins** [comment]
Spacing is a little tight for me, strangely (in both FF and Chrome on Windows):

image

**Comment by thybag** [comment]
Just wondering if its worth also removing the settings icon from the "media control bar", given its moving to the site attribution etc?

**Comment by andytuba** [comment]
I'd rather keep both since the attribution link goes directly to the particular site module settings while the control bar links to its particular setting.. and because users are myopic.

File: lib/core/migrate.js [hunk 1]
```diff
@@ -262,6 +262,54 @@ const migrations = [
 		async go() {
 			await migrators.generic.updateOption('singleClick', 'hideLEC', false, true);
 		},
+	}, {
+		versionNumber: '4.7.0-showImages-siteModuleIDs',
+		async go() {
+			await migrators.generic.moveOption('styleTweaks', 'display uploadly', 'styleTweaks', 'display_uploadly');
```
Comment (erikdesjardins) [inline lib/core/migrate.js]: This should be `showImages` right?

File: lib/templates/searchResultOption.mustache [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <div class="SearchRES-result-header">
     <span class="SearchRES-result-title">{{title}}</span>
-				<span class="SearchRES-breadcrumb">RES settings console
-					&gt; {{category}}
-                    &gt; {{moduleName}} ({{moduleID}})
-					{{#optionKey}} &gt; {{optionKey}}{{/optionKey}}
-				</span>
+	<span class="SearchRES-breadcrumb">RES settings console
+		&gt; {{category}}
+        &gt; {{moduleName}} ({{moduleID}})
```
Comment (erikdesjardins) [inline lib/templates/searchResultOption.mustache]: Spacing is a little off here

File: lib/core/migrate.js (hunk 1 above)
Comment (andytuba) [inline lib/core/migrate.js]: yyyyyyyep

**Pull request merged**